spark/src/main/scala/org/apache/spark/sql/delta/Checksum.scala

```diff
@@ -54,6 +54,7 @@ case class VersionChecksum(
     numFiles: Long,
     numMetadata: Long,
     numProtocol: Long,
+    @JsonDeserialize(contentAs = classOf[Long])
```

Comment:
need `import com.fasterxml.jackson.databind.annotation.JsonDeserialize`?

Reply:
Thanks for catching that! Fixed.